janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1150,6 +1150,40 @@ public void testGraphCentricQueryProfiling() {
                 put("index_impl", "search");
             }};
             assertEquals(annotations, nested.getAnnotations());
+
+            // multiple or clause satisfied by a single graph-centric query which is satisfied by a single mixed index query
```

Comment:
Can you add a case where the conditions are satisfied by the union of two mixed index queries (each of which covers a "or" query)?

Reply:
added